### PR TITLE
Green unit tests in pkg/build

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: pkg/build/builder/docker_test.go [hunk 1]
```diff
@@ -193,7 +192,7 @@ func TestDockerfilePath(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		buildDir, err := ioutil.TempDir(util.GetBaseDir(), "dockerfile-path")
```
Comment (ncdc) [inline pkg/build/builder/docker_test.go:196]: Why this change?

File: pkg/build/builder/docker_test.go [hunk 2]
```diff
@@ -339,7 +339,7 @@ RUN echo "hello world"
 		},
 	}
 	for i, test := range tests {
-		buildDir, err := ioutil.TempDir(util.GetBaseDir(), "dockerfile-path")
```
Comment (ncdc) [inline pkg/build/builder/docker_test.go:342]: ditto

**Comment by ncdc** [comment]
@sttts just 1 question, then lgtm

File: pkg/build/builder/docker_test.go (hunk 1 above)
Comment (sttts) [reply pkg/build/builder/docker_test.go:196]: `util.GetBaseDir()` does not exist. `TempDir` does not create it.

**Comment by sttts** [comment]
Merging, ok?

**Comment by ncdc** [comment]
Merge away
